vm/vm.go

```diff
@@ -382,7 +387,8 @@ func (mon *monitor) appendOutput(out []byte) *report.Report {
 func (mon *monitor) extractError(defaultError string) *report.Report {
 	if mon.finished != nil {
 		// If the caller wanted an early notification, provide it.
-		mon.finished()
+		mon.finished(true)
```

Comment:
This is going to give a false positive (say that it crashed when in fact it didn't) only for the case of GCE preemption, right?

Reply:
Good question. It will probably be more reliable if we keep it as it is now.

I've pushed a different commit instead that always calls the callback so that it's possible to rely on it being called (+fixes a bug).

PTAL